### ISSUE TITLE
CLI default options now removing tag prefixes

### ISSUE THREAD

**Issue by deimyts** [conventional-changelog/conventional-github-releaser#44]
I'm using a `v` prefix for my version tags (`v1.0.0`), and 
I'm using `conventional-github-releaser` from the CLI.

After tagging the repo, I run the following commands:
`git push --follow-tags origin master`
`conventional-github-releaser -t $GITHUB_TOKEN -p angular` 

If my tag is `v1.0.1`, instead of creating the release on that tag, `conventional-github-releaser` is adding a new tag to my remote repo, without the prefix: `1.0.1`. It then attaches the release to _that_ tag, instead of the one that I actually pushed. It doesn't delete the prefix tag, it just fails to use it.

This behavior started sometime between `1.1.3` and `1.1.9`

**Comment by egoldblum** [comment]
I have the exact same workflow and the exact same issue.

It appears that this project's own releases also went from havinga  `v` prefix to not having one between `v1.1.7` and `1.1.8`.

https://github.com/conventional-changelog/conventional-github-releaser/releases

**Comment by fergiemcdowall** [comment]
+1

**Comment by fergiemcdowall** [comment]
Can confirm: works as expected on `1.1.3`. On all subsequent versions, "v"-prefixes are mysteriously stripped away.

**Comment by hbetts** [comment]
I'm pretty sure the missing `v` is a result of [this change](https://github.com/conventional-changelog/conventional-github-releaser/commit/b92b06463cff161ae54c1df99b22774703303c7d).

That change switched to using `find-versions`, which strips the leading `v`.

I'll continue looking into the issue later, but in the meantime does anyone have any suggestion for a package that can extract semantic valid versions, while keeping the leading `v` in place?

**Comment by hbetts** [comment]
I've published version [v1.1.10](https://github.com/conventional-changelog/conventional-github-releaser/releases/tag/v1.1.10) which contains a fix to the `transform` function.

The `transform` function will no longer remove the leading `v` in a tag.

Full details available in the commit message - 1b6a68cf433fe90836aa1ed3b2961d39dd035732

I've also added additional unit tests to check for the desired behavior and prevent future regression.

In closing, I'm really sorry for the bug I introduced, and I hope y'all will continue to use `conventional-github-releaser`, and get value from it.
